third_party/move/move-core/types/src/function.rs

```diff
@@ -156,15 +156,37 @@ impl ClosureMask {
         i
     }
 
-    pub fn merge_placeholder_strings(
-        &self,
-        arity: usize,
-        captured: Vec<String>,
-    ) -> Option<Vec<String>> {
-        let provided = (0..arity - captured.len())
-            .map(|_| "_".to_string())
-            .collect::<Vec<_>>();
-        self.compose(captured, provided)
+    pub fn merge_placeholder_strings(&self, captured: Vec<String>) -> Vec<String> {
```

Comment:
Could we please add a doc string to this public function?

Reply:
Added, also renamed to `format_arguments` so its clearer